### ISSUE TITLE
Clojure code snippets don't have syntax highlighting

### ISSUE THREAD

**Issue by elektronik2k5** [freeCodeCamp/devdocs#2216]
# Bug report

Clojure code snippets are missing syntax highlighting.

## Steps to reproduce

1. Enable Clojure
2. Open a page with a code snippet

## More resources

Expected outcome: code snippets should have syntax highlighting.
 

## Possible fix

Add syntax highlighting or fix whatever prevents it from working with Clojure source code samples.

**Comment by thewheat** [comment]
Hi @elektronik2k5 , could you link to a page that shows the syntax highlighting? 

I've tried looking at https://clojure.github.io/clojure/ and several pages and I'm not seeing any syntax highlighting specifically, so perhaps I'm missing something

**Comment by elektronik2k5** [comment]
@thewheat not sure I understand what you're asking for: any webpage that includes Clojure code with syntax highlighting?

**Comment by thewheat** [comment]
Correct. I couldn't find any syntax highlighting in the source docs so wasn't sure what you were referring to exactly. 

Or is it more about adding syntax highlighting that isn't even in the source documentation?
